src/templates/static/mt5_signals.jsx

```diff
@@ -0,0 +1,238 @@
+                                            <li> {it.L('You won’t be able to trade manually with the same [_1] MT5 account once you’ve subscribed to a signal.', it.website_name)}
+                                            </li>
+                                        </ol>
+                                        <p>{it.L('Have more questions? Check out the [_1]FAQ[_2] section on the MQL5 website.', '<a href="https://www.mql5.com/en/forum/10773#q17">', '</a>')} </p>
+                                        <div className='center-text gr-padding-20'>
+                                            <a className='button' href={it.url_for('user/metatrader')}><span>{it.L('Go to the MT5 Terminal')}</span></a>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+
+                        </TabContent>
+                        <TabContent id='signal_provider'>
+                            <div className='container gr-padding-20 gr-parent'>
+                                <p className='text-align-center'>
+                                    {it.L('If you are a professional trader, the MT5 trading signals service allows you to share your strategies with other traders for free or a subscription fee that you determine. When traders subscribe to your signal, your deals are automatically replicated on their accounts each time you place a trade.')}
+                                </p>
+                            </div>
+
+                            <div className='fill-bg-color gr-padding-30 fullwidth' >
+                                <div className='container gr-padding-30 gr-child'>
+                                    <h2 className='center-text'>
+                                        {it.L('Benefits of being an MT5 provider')}
+                                    </h2>
+                                    <div className='fill-bg-color gr-padding-30'>
+                                        <div className='container section'>
+                                            <div className='gr-padding-10 facts'>
+                                                <div className='gr-row no-padding'>
+                                                    <Box>
+                                                        <BoxInner className='border-right-top' image='easy_setup' text={it.L('Easy to set up – no installation required.')} />
+                                                        <BoxInner className='border-right-top' image='additional_income' text={it.L('Additional income stream from monthly subscriptions')} />
+                                                        <BoxInner className='border-right-top' image='automated_copy_trade' text={it.L('Automated copying – no extra work from you')} />
```

Comment:
the first has a `.` at the end of the sentence but the next two don't, is that intended?

Reply:
These are from designs , but I guess adding `.` makes more sense

Reply:
okay, please ask them to update the design too